permissions_validators/src/private_blockchain/query.rs

```diff
@@ -86,6 +86,21 @@ impl<W: WorldTrait> IsAllowed<W, QueryBox> for OnlyAccountsDomain {
                         )
                     })?
             }
+            FindTriggersByDomainId(query) => {
+                let domain_id = query
+                    .domain_id
+                    .evaluate(wsv, &context)
+                    .map_err(|e| e.to_string())?;
+
+                if domain_id == authority.domain_id {
+                    return Ok(());
+                }
+
+                Err(format!(
+                    "Cannot access triggers with given domain {}, {} is permitted..",
```

Comment:
the second half of the sentence is unclear
do you mean that with the `domain_id` you can't access triggers, but if you use `authority.domain_id` you'll be able to?

Reply:
yes